DOCSP-16785 the the typo (#3297)
mongodb/docs-realm: temp/cpp/users/custom-user-data.txt

Custom User Data - C++ SDK
==========================

.. meta:: 

>   :description: Learn how to create custom user data using Atlas Device SDK for C++.

Reviewer: ```suggestion
  :description: Learn how to create custom user data using the Atlas Device SDK for C++.
```
nit

>   :description: Learn how to create custom data for a user using Atlas App Services in C++

Reviewer: ```suggestion
  :description: Learn how to create custom user data using Atlas Device SDK for C++.
```
